src/particles/feedback_CIC_gpu.cu

```diff
@@ -254,14 +275,15 @@ __global__ void Cluster_Feedback_Kernel(
     pos_x = pos_x_dev[gtid];
     pos_y = pos_y_dev[gtid];
     pos_z = pos_z_dev[gtid];
-    // printf("(%d): pos:(%.4e, %.4e, %.4e)\n", gtid, pos_x, pos_y, pos_z);
-    // printf("(%d): MIN:(%.4e, %.4e, %.4e)\n", gtid, xMin, yMin, xMin);
+    // kernel_printf("(%d): pos:(%.4e, %.4e, %.4e)\n", gtid, pos_x, pos_y,
+    // pos_z); kernel_printf("(%d): MIN:(%.4e, %.4e, %.4e)\n", gtid, xMin, yMin,
+    // xMin);
 
     bool in_local = (pos_x >= xMin && pos_x < xMax) &&
                     (pos_y >= yMin && pos_y < yMax) &&
                     (pos_z >= zMin && pos_z < zMax);
     if (!in_local) {
-      printf(
+      kernel_printf(
```

Comment:
What's the purpose of all the `printf`s?

Reply:
Not my purview. The scope of the changes here was only to rename some printfs within kernels with an easy to way to find them all.

Reply:
Fair enough.